src/app.ts

```diff
@@ -0,0 +1,24 @@
+import express, { Application }  from 'express';
+import { validationErrorHandler } from "./ValidationErrorHandler/errorHandler";
+
+const app: Application = express();
+const homeRoute = require('./routes/home');
+const usersRoute = require('./routes/users');
+const mongoose = require('mongoose');
+ 
+const port: number = 8080;
+const mongodbUrl: string = 'mongodb+srv://user:214365879@cluster0.w2spu.mongodb.net/myFirstDatabase?retryWrites=true&w=majority';

```

Comment:
The better way to extract this to app config, for this goal you can use `config` package or `dotenv` (don't know which exactly, I used `config`)

Reply:
100% agree, I'm planning to beautify folder and file structure in the next module according 3-layer architecture principles.